packages/braid-design-system/package.json

```diff
@@ -171,6 +171,7 @@
     "@types/node": "^18.13.0",
     "@types/react": "^18.0.21",
     "@types/react-dom": "^18.0.6",
+    "@types/react-is": "^17.0.3",
```

Comment:
Confirming, is this version correct? Not sure if the definitely typed package is just out of step with version numbers

Reply:
Yep, that's the most recent [version](https://www.npmjs.com/package/@types/react-is).